packages/playground/_stories/main/Panel/Panel.stories.ts

```diff
@@ -23,77 +28,72 @@ export default {
-			<h1 class="content-color">I am a native heading!</h1>
-			<ui5-label wrapping-type="Normal">Short text.</ui5-label>
-			<br/>
-			<ui5-label wrapping-type="Normal">Another text.</ui5-label>
-			<p class="content-color">
-				Aute ullamco officia fugiat culpa do tempor tempor aute excepteur magna. Quis velit adipisicing excepteur do eu duis elit. Sunt ea pariatur nulla est laborum proident sunt labore commodo Lorem laboris nisi Lorem.
-			</p>
-		</ui5-panel>
-	</div>
-`;
+const Template : UI5StoryArgs<Panel, StoryArgsSlots> = (args) => html`
+<ui5-panel
+	accessible-role="${ifDefined(args.accessibleRole)}"
+	header-text="${ifDefined(args.headerText)}"
+	?fixed="${ifDefined(args.fixed)}"
+	?collapsed="${ifDefined(args.collapsed)}"
+	?no-animation="${ifDefined(args.noAnimation)}"
+	header-level="${ifDefined(args.headerLevel)}"
+	accessible-name="${ifDefined(args.accessibleName)}"
+>
+	${unsafeHTML(args.header)}
+	${unsafeHTML(args.default)}
+</ui5-panel>`;
 
+export const BasicPanel = Template.bind({});
+BasicPanel.args = {
+	default: '<h1 class="content-color">I am a native heading!</h1><ui5-label wrapping-type="Normal">Short text.</ui5-label><br/><ui5-label wrapping-type="Normal">Another text.</ui5-label><p class="content-color">Aute ullamco officia fugiat culpa do tempor tempor aute excepteur magna. Quis velit adipisicing excepteur do eu duis elit. Sunt ea pariatur nulla est laborum proident sunt labore commodo Lorem laboris nisi Lorem.</p>',
+	headerText: "Both expandable and expanded",
+	fixed: false,
+	collapsed: false,
+	noAnimation: false,
+	accessibleRole: PanelAccessibleRole.Complementary,
+	headerLevel: TitleLevel.H2,
+	accessibleName: "",
+	header: ""
+};
+BasicPanel.storyName = "Basic Panel";
 
-export const Template1: StoryFn = () => html`
-<h3>Panel with List</h3>
-	<div class="snippet">
-		<ui5-panel accessible-role="Complementary" header-text="Select your country" class="full-width">
-			<ui5-list id="myList1" mode="MultiSelect">
-				<ui5-li key="country1">Argentina</ui5-li>
-				<ui5-li key="country2">Bulgaria</ui5-li>
-				<ui5-li key="country3">China</ui5-li>
-				<ui5-li key="country4">Germany</ui5-li>
-				<ui5-li key="country5">Hungary</ui5-li>
-				<ui5-li key="country6">England</ui5-li>
-				<ui5-li key="country7">USA</ui5-li>
-				<ui5-li key="country8">Canada</ui5-li>
-			</ui5-list>
-		</ui5-panel>
-	</div>
-`;
+export const PanelWithList = Template.bind({});
+PanelWithList.args = {
+	default: '<ui5-list mode="MultiSelect"><ui5-li key="country1">Argentina</ui5-li><ui5-li key="country2">Bulgaria</ui5-li><ui5-li key="country3">China</ui5-li><ui5-li key="country4">Germany</ui5-li><ui5-li key="country5">Hungary</ui5-li><ui5-li key="country6">England</ui5-li><ui5-li key="country7">USA</ui5-li><ui5-li key="country8">Canada</ui5-li></ui5-list>',
+	headerText: "Select your country",
+	accessibleRole: PanelAccessibleRole.Complementary
+};
+PanelWithList.storyName = "Panel with List";
```

Comment:
no need of the story name as the story variable name will result with the same

Reply:
Done